### ISSUE TITLE
implement get_confusion_matrix method in python

### ISSUE THREAD

**Issue by billy-odera** [catboost/catboost#802]
I am unable to plot the ROC - AUC graph in catboost. The code displays the ROC -AUC value but I need the graph
```
from sklearn.metrics import roc_auc_score, roc_curve, f1_score, make_scorer
pred = clf_catboost.predict_proba(test_X)[:,1]
print( roc_auc_score(test_Y, pred))
tpr,fpr,_=roc_curve(y_score=pred,y_true=test_Y)
```

**Also, how can we get the confusion matrix.

These are important when using catboost for credit scoring**

**Comment by nikitxskv** [comment]
We added parameter `plot` to `get_roc_curve` method, so you can use it like this:
```
from catboost.utils import get_roc_curve
from catboost import Pool

get_roc_curve(clf_catboost, Pool(test_X, test_Y), plot=True)
```

This change isn't on pypi now (it will happen at next week), but you can build catboost from sources:
https://catboost.ai/docs/installation/python-installation-method-build-from-source-linux-macos.html

**Comment by annaveronika** [comment]
Implemented in catboost 0.16.3
